<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFEFCSSimMainWidnow</class>
 <widget class="QWidget" name="QFEFCSSimMainWidnow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FCS Simulator (&quot;diffusion4&quot;)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qfe_fcssimulator.qrc">
    <normaloff>:/qfe_fcssimulator/qfe_fcssimulator.png</normaloff>:/qfe_fcssimulator/qfe_fcssimulator.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="toolTip">
      <string>choose the number of parallel simulator taks on this machine</string>
     </property>
     <property name="title">
      <string> Configuration </string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Execution Directory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFEnhancedLineEdit" name="edtExeDir">
        <property name="toolTip">
         <string>choose the execution directory of the simulator. relative paths in a script will be generated relative to this directory.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Simulator:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFEnhancedLineEdit" name="edtSimulator">
        <property name="toolTip">
         <string>choose the simulator executable (you can choose a custom version here, if you have one!)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Simulator: spectra database:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFEnhancedLineEdit" name="edtSimulatorSpectra">
        <property name="toolTip">
         <string>choose the directory with the simulatoir fluorophore database</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>max. parallel processes:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spinMaxProcesses">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkSaveLog">
          <property name="toolTip">
           <string>if checked, a &lt;tt&gt;script.ini.log&lt;/tt&gt; with the log-output of the simulator will be generated in the execution directory.</string>
          </property>
          <property name="text">
           <string>save log-file in exe-directory</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnClose">
        <property name="toolTip">
         <string>Close the simulator window</string>
        </property>
        <property name="text">
         <string>&amp;Close</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/lib/exit.png</normaloff>:/lib/exit.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnNew">
        <property name="toolTip">
         <string>create a new simulator configuration</string>
        </property>
        <property name="text">
         <string>&amp;New</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/lib/icons/document_new.png</normaloff>:/lib/icons/document_new.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnNewFromTemplate">
        <property name="toolTip">
         <string>create a new simulator configuration from a template</string>
        </property>
        <property name="text">
         <string>New From &amp;Template</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/lib/icons/document_import.png</normaloff>:/lib/icons/document_import.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnOpen">
        <property name="toolTip">
         <string>open an existing simulator configuration script ...</string>
        </property>
        <property name="text">
         <string>&amp;Open</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/lib/icons/document_open.png</normaloff>:/lib/icons/document_open.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSave">
        <property name="toolTip">
         <string>save the current script</string>
        </property>
        <property name="text">
         <string>&amp;Save</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/lib/icons/document_save.png</normaloff>:/lib/icons/document_save.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSaveAs">
        <property name="toolTip">
         <string>save the current script under a new name</string>
        </property>
        <property name="text">
         <string>Save &amp;as</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/lib/icons/document_save_all.png</normaloff>:/lib/icons/document_save_all.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnExecute">
        <property name="toolTip">
         <string>execute the current simulator configuration script</string>
        </property>
        <property name="text">
         <string>Execute</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/lib/wizard.png</normaloff>:/lib/wizard.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnStop">
        <property name="toolTip">
         <string>cancel the execution (if it is running) in the current simulator script tab</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/lib/unchecked.png</normaloff>:/lib/unchecked.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btnHelp">
        <property name="toolTip">
         <string>open the plugin help window</string>
        </property>
        <property name="text">
         <string>&amp;Help</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/lib/help/help.png</normaloff>:/lib/help/help.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnManual">
        <property name="toolTip">
         <string>open the PDF-file with the simulator manual</string>
        </property>
        <property name="text">
         <string>simulator manual</string>
        </property>
        <property name="icon">
         <iconset resource="../../lib/lib.qrc">
          <normaloff>:/pdf.png</normaloff>:/pdf.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFEnhancedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfenhancedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="qfe_fcssimulator.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
 </resources>
 <connections/>
</ui>
